.circleci/test.sh

```diff
@@ -14,6 +14,18 @@ if [ -z "$TERMINUS_SITE" ] || [ -z "$TERMINUS_ENV" ]; then
 	exit 1
 fi
 
+if [ -z "$TERMINUS_SITE" ] || [ -z "$TERMINUS_ENV" ]; then
+	echo "TERMINUS_SITE and TERMINUS_ENV environment variables must be set"
+	exit 1
+fi
+
+# Check for rejection file and exit if once exists
+rej_file=`ls -1 *.rej 2>/ | wc -l`
```

Comment:
We'll want to use `find` rather than `ls` to ensure that we find .rej files anywhere in the source tree.